File: expresslk-services/ExpressLK-core/src/main/java/lk/express/cnx/CancellationCause.java
package lk.express.cnx;

import javax.xml.bind.annotation.XmlRootElement;
import javax.xml.bind.annotation.XmlType;

@XmlType(name = "CancellationCause", namespace = "http://cnx.express.lk")
@XmlRootElement
public enum CancellationCause {
	ClientRequested, NonPayment, DataEntryError, Other
}

File: expresslk-webportal/module/Api/src/Api/Client/Soap/Core/CancellationCause.php
<?php

namespace Api\Client\Soap\Core;

class CancellationCause
{
    const __default = 'ClientRequested';
    const ClientRequested = 'ClientRequested';
    const NonPayment = 'NonPayment';
    const DataEntryError = 'DataEntryError';
    const Other = 'Other';


}
